consensus/types/messages.go

```diff
@@ -155,13 +63,13 @@ const (
 	selfProposalError                           = "hotstuff message is a self proposal"
 	olderStepRoundError                         = "hotstuff message is of the right height but from the past"
 	unexpectedPacemakerCaseError                = "an unexpected pacemaker case occurred"
-	invalidAppHashError                         = "apphash being applied does not equal that from utility"
-	byzantineOptimisticThresholdError           = "byzantine optimistic threshold not met"
+	invalidStateHashError                       = "stateHash being applied does not equal that from utility"
+	byzantineOptimisticThresholdError           = "BFT threshold not met"
 	consensusMempoolFullError                   = "mempool is full"
 	applyBlockError                             = "could not apply block"
 	prepareBlockError                           = "could not prepare block"
 	commitBlockError                            = "could not commit block"
-	replicaPrepareBlockError                    = "node should not call `prepareBlock` if it is not a leader"
+	replicaPrepareBlockError                    = "REPLICATE SHOULD NOT call `prepareBlock`"
```

Comment:
Behave @Olshansk! :) Not sure if you want to "shout", looking at the other errors, we generally don't

```suggestion
	replicaPrepareBlockError                    = "replica should not call `prepareBlock`"
```

Reply:
?? Just wondering if you missed a commit or something. This comment is marked as resolved but I cannot see the change